source/guides/first-steps/task-script.md

```diff
@@ -1,4 +1,127 @@
-Your First Task Script (NOT YET WRITTEN)
+Your First Task Script
 ----------------------
 
-**TODO: Write-up that guides the user through writing a basic task script (just a task with a narrate line) using VS-Code, emphasizing things like file extension and placement. Also run it in-game with `/ex run`.**
+<!-- **TODO: Write-up that guides the user through writing a basic task script (just a task with a narrate line) using VS-Code, emphasizing things like file extension and placement. Also run it in-game with `/ex run`.** -->
+
+```eval_rst
+.. contents:: Table of Contents
+    :local:
+```
+
+## Task script basics
+
+A task script is a stand-alone script that can be run either via in-game command with the `/ex` command, as explained in [the guide to the /ex command](https://guide.denizenscript.com/guides/first-steps/ex-command.html).
```

Comment:
Broken sentence. Forgot `by the "run" command in a script or` in the middle?

Reply:
lol yes